src/main/java/org/pih/petl/job/type/PentahoJob.java

```diff
@@ -87,6 +86,7 @@ public void execute(final ExecutionContext context) throws Exception {
             try {
                 log.debug("Initializing Kettle Environment");
                 log.debug("KETTLE_HOME = " + System.getProperty("KETTLE_HOME"));
+                System.setProperty("KETTLE_PLUGIN_CLASSES", "org.pentaho.di.trans.steps.append.AppendMeta");
```

Comment:
This is the class that was moved to a plugin in pentaho 9, and this is what you need to do to tell Kettle to load it.